apps/i18n/music/en_us.json

```diff
@@ -1,2 +1,12 @@
 {
+  "arpeggioUp": "Arpeggio Up",
+  "arpeggioDown": "Arpeggio Down",
+  "arpeggioRandom": "Arpeggio Random",
+  "control": "Control",
+  "feedback": "Feedback",
+  "share": "Share",
+  "shareComingSoon": "Sharing is under construction. Check back soon.",
+  "startOver": "Start Over",
+  "together": "Together",
```

Comment:
Curious whether there's a way for us to attach a description so that translators know the context that this is to "play together/simultaneously".  Otherwise, this one in particular feels pretty ambiguous.

Reply:
Good point. Maybe "chordTogether", as well as "chordArpeggioUp" for the others, to be clear and show that they're associated?

Reply:
That's a good idea.  Also, a common prefix will help group them, which we don't always need for localized strings when the key relates more to the text itself (to keep the code kind of readable) but makes quite a bit of sense here where they are all options in the same dropdown.